laterpay/application/Controller/Install.php

```diff
@@ -493,4 +492,31 @@ public function update_capabilities( LaterPay_Core_Event $event ) {
         $laterpay_capabilities = new LaterPay_Core_Capability();
         $laterpay_capabilities->update_roles( (array) $roles );
     }
+
+    /**
+     * Update Default Pricing Behaviour to Provide Backwards Compatibility
+     *
+     * @since 2.1.0
+     */
+    public function update_default_pricing_behaviour() {
+        $current_version = get_option( 'laterpay_plugin_version' );
+        if ( version_compare( $current_version, '2.0.0', '<' ) ) {
+            return;
+        }
+
+        if ( false === get_option( 'laterpay_post_price_behaviour' ) ) {
+            $only_timepass        = (bool) get_option( 'laterpay_only_time_pass_purchases_allowed' );
+            $current_global_price = ( float) get_option( 'laterpay_global_price' );
+
+            if ( true === $only_timepass ) {
+                update_option( 'laterpay_post_price_behaviour', 1 );
+            } else if ( floatval( 0.00 ) === $current_global_price ) {
```

Comment:
We have a mix of `else if` and `elseif` in code base we can clean that up later on. No action needed right now on this.